CONTRIBUTING.md

```diff
@@ -1,32 +1,103 @@
-The Janitor sits atop a number of other projects, and those are where
-most of the interesting things happen. You may want to check out one of them.
+[The Janitor](https://github.com/jelmer/janitor) sits atop a number of other
+projects, and those are where most of the interesting things happen.
+You may want to check out one of them.
 They're probably also easier to setup and run, unlike the Janitor.
 
-Environment
-===========
+## Environment
 
-Mostly you can use pip to install Python-based dependencies. In addition to
-those, you'll also want to install various other bits of software.
+### Development
+
+It is recommended to use the latest [Debian](https://tracker.debian.org/pkg/rustc)
```

Comment:
Please use debian unstable or testing - that's what the janitor has historically been using. The goal is to allow packaging in Debian, so I'd rather avoid recommending rustup.

Reply:
Noted.
I've updated the wording.